pkg/datastore/test/basic.go

```diff
@@ -83,3 +83,21 @@ func DeleteAllDataTest(t *testing.T, tester DatastoreTester) {
 		}
 	}
 }
+
+func UniqueIDTest(t *testing.T, tester DatastoreTester) {
+	require := require.New(t)
+
+	// Create the datastore.
+	ds, err := tester.New(0, veryLargeGCInterval, veryLargeGCWindow, 1)
+	require.NoError(err)
+
+	// Ensure the unique ID is not empty.
+	uniqueID, err := ds.UniqueID(context.Background())
+	require.NoError(err)
+	require.NotEmpty(uniqueID)
+
+	// Ensure the unique ID is stable.
```

Comment:
how is this code testing that it's stable across restarts? where is the restart happening? (it seems that every test is spinning up a new DS but not stopping it?)

Reply:
It only verifies it is stable across calls; it is not verifying across restarts (nor does any other data storage test, mind you... that is assumed)